wooey/tests/test_forms.py

```diff
@@ -72,3 +72,14 @@ def test_multiplechoice_form(self):
         file_param = 'multiple_file_choices'
         files = [i.value for i in job.get_parameters() if i.parameter.slug == file_param]
         self.assertEqual(len(files), len(fdict.get(file_param)))
+
+    def test_without_args_form(self):
+        script_version = self.without_args
+        form = utils.get_master_form(script_version=script_version)
+        # check our wrapper is in the form render
+        utils.validate_form(form=form)
+        self.assertTrue(form.is_valid())
```

Comment:
Can you add an assertment stating the form's pk is = to the script's pk. Since that is the change made above (wooey_type). I think this is the crux here